reference/catalog/category-trees_catalog.v3.yml

```diff
@@ -224,6 +224,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorRequest'
+        '409':
+          description: 'A category with the same name already exists within the given category tree.'
```

Comment:
It could also be returned if category with the same name already exist within the given parent category. I'd better frame it as `A category with the same name already exists within the given category tree or within the given parent category.`